src/provider/provider_tracking.c

```diff
@@ -70,7 +70,8 @@ static umf_result_t umfMemoryTrackerRemove(umf_memory_tracker_handle_t hTracker,
 
     void *value = critnib_remove(hTracker->map, (uintptr_t)ptr);
     if (!value) {
-        LOG_ERR("umfMemoryTrackerRemove: Unknown Error");
+        LOG_ERR("umfMemoryTrackerRemove: pointer not found in the map: %p",
```

Comment:
this message is a little confusing (p is the address of map or ptr?), " pointer  %p" not found in the map" would be more precise

Reply:
Done